tests/test_main.py

```diff
@@ -113,6 +115,9 @@ def test_create_suite(get_data, get_output):
     check_command_result(result)
     assert output_path.exists()
 
+    suite = JsonParser.parse_object(output_path, SuiteABC)
+    assert len(suite.required_tests) > 0
```

Comment:
This assertion fails with the original version but passes with the new version. I want to make it a habit to add tests whenever bugs surface to prevent regression.